README.md

```diff
@@ -1,3 +1,17 @@
+## Contents
```

Comment:
thank you SO MUCH for adding this!! I was planning on putting in a PR for it! This is great 🎉